oap-server/server-library/library-client/src/main/java/org/apache/skywalking/oap/server/library/client/elasticsearch/ElasticSearchClient.java

```diff
@@ -119,20 +123,20 @@ public boolean createIndex(String indexName, JsonObject settings, JsonObject map
         return response.isAcknowledged();
     }
 
-    public List<String> retrievalIndexByAliases(String aliases) throws IOException {
+    public List<ElasticSearchTimeSeriesIndex> retrievalIndexByAliases(String aliases) throws IOException {
         aliases = formatIndexName(aliases);
-
         Response response = client.getLowLevelClient().performRequest(HttpGet.METHOD_NAME, "/_alias/" + aliases);
-
-        List<String> indexes = new ArrayList<>();
         if (HttpStatus.SC_OK == response.getStatusLine().getStatusCode()) {
             Gson gson = new Gson();
             InputStreamReader reader = new InputStreamReader(response.getEntity().getContent());
             JsonObject responseJson = gson.fromJson(reader, JsonObject.class);
             logger.debug("retrieval indexes by aliases {}, response is {}", aliases, responseJson);
-            indexes.addAll(responseJson.keySet());
+            Set<String> keySet = responseJson.keySet();
+            if (!keySet.isEmpty()) {
+                return keySet.stream().map(this::undoFormatIndexName).map(index -> new ElasticSearchTimeSeriesIndex(namespace, index)).collect(Collectors.toList());
+            }
         }
-        return indexes;
+        return Collections.EMPTY_LIST;
     }
 
     public JsonObject getIndex(String indexName) throws IOException {
```

Comment:
`getIndex` method needs to move to `ITElasticSearchClient` with `undoFormatIndexName(JsonObject index)` method.

Reply:
done